cppfd/solver.cpp

```diff
@@ -2,41 +2,114 @@
 
 #include"solver.h"
 
-Solver::Solver(Mesh& m, double d_t, double t_f, double r, double d_v, double m_C, int v)
-  : mesh(m)
-  , delta_t(d_t)
-  , t_final(t_f)
-  , rho(r)
-  , max_C(m_C)
-  , verbosity(v)
-  {
-    // compute and store kinematic viscosity
-    this->nu = d_v / r;
+// main function to solve N-S equation on time steps
+void Solver::solve(){
+  // begin simulation
+  if(this->verbosity >= 1){
+    std::cout<<"Initial timestep : " << this->delta_t << ", " << "Time final : " << this->t_final <<std::endl;
   }
 
-// main function to solve N-S equation on time steps
-void Solver::solve(BoundaryConditions& b_c)
-{
-  // implement me
+  Kokkos::Timer timer;
+
+  if(this->verbosity >= 1){
+    std::cout<<"Applying velocity boundary conditions..."<<std::endl;
+  }
+  this->boundary_conditions.apply_velocity_bc();
+
+  double velocity_bc_max = this->boundary_conditions.get_velocity_bc_max_norm();
```

Comment:
Add comment here to explain that this is the computation of the Reynolds number (not a primary input to the code)